package.json

```diff
@@ -38,8 +38,10 @@
     "test:playwright:update-tapes": "UPDATE_TAPES=true pnpm test:playwright",
     "test:playwright:gen": "playwright codegen localhost:8443/",
     "types": "vue-tsc --noEmit",
-    "lint": "eslint --ext .js,.vue,.ts --ignore-path .gitignore --ignore-path .eslintignore . && prettier --check .",
-    "lint:fix": "pnpm lint -- --fix && prettier --write --loglevel=warn .",
```

Comment:
Maybe replacing `&&` with `;` would have sufficed? That would allow both commands to run regardless if one failed.

Reply:
It certainly continues even if it fails but that way does not apply eslint fixes and shows error as output that prettier fixed, so it's a bit confusing/annoying because one might think there are still errors when is not the case.

Reply:
Would reversing the order and running prettier first _then_ eslint also fix this problem without adding new scripts?

In part of Dhruv's PR we discussed trying to pair down the scripts in package.json to ones that are truly useful (mostly reusable), otherwise it adds a lot of noise. Individual eslint and prettier scripts were something that ended up being avoided because you never want to run one without the other.

Reply:
@sarayourfriend that works! Done in [`35b8a19`](https://github.com/WordPress/openverse-frontend/pull/1239/commits/35b8a199e723956fc3d5e5ff370e68b333e81173). I don't mind much two extra lines of scripts and this solution execute prettier check two times but it's not a big burden either 🤷🏻‍♀️

Reply:
@sarayourfriend that works! Done in [`35b8a19`](https://github.com/WordPress/openverse-frontend/pull/1239/commits/35b8a199e723956fc3d5e5ff370e68b333e81173). I don't mind much two extra lines of scripts and this solution execute prettier check two times but it's not a big burden either 🤷🏻‍♀️